test/remote/usps_test.rb

```diff
@@ -125,6 +125,70 @@ def test_international_rates
     assert_equal [], other_than_two, "Some RateEstimates do not refer to the right number of packages (#{other_than_two.inspect})"
   end
 
+  def test_domestic_label_from_name_errros
+    assert_raises(ResponseError, "From Name is required and may not exceed 100 characters.") do
+      @carrier.create_shipment(
+        location_fixtures[:new_york],
+        location_fixtures[:beverly_hills_with_name],
+        package_fixtures.values_at(:book),
+        :test => true,
+        :service => :first_class
+      )
+    end
+  end
+
+  def test_domestic_label_to_name_errors
+    assert_raises(ResponseError, "To Name is required and may not exceed 100 characters.") do
+      @carrier.create_shipment(
+        location_fixtures[:new_york_with_name],
+        location_fixtures[:beverly_hills],
+        package_fixtures.values_at(:book),
+        :test => true,
```

Comment:
Wanted to follow the style of the entire project here, I can open up a separate PR which updates all hash styles.